Domains/Profiles/Profiles.ecschema.xml

```diff
@@ -366,4 +367,118 @@
         <ECProperty propertyName="Width" typeName="double" description="Extent of the capsule in the direction of the x-axis." category="ProfileProperties" kindOfQuantity="AECU:LENGTH_SHORT" />
         <ECProperty propertyName="Depth" typeName="double" description="Extent of the capsule in the direction of the y-axis." category="ProfileProperties" kindOfQuantity="AECU:LENGTH_SHORT" />
     </ECEntityClass>
+
+    <ECEntityClass typeName="ProfiledSolidDefinitionElement" modifier="Abstract" displayLabel="Profiled Solid Definition Element" description="TODO">
```

Comment:
I think ProfiledSolidDefinitionElement and ProfiledSolidDefinition classes are named almost same.
Is it possible to rename these? 
e.g:

- ProfiledSolidDefinitionElement -> ProfiledSolidDefinition
- ProfiledSolidDefinition -> UniformProfiledSolidDefinition

Reply:
It's been a convention in BIS to use the "Element" suffix for abstract element classes. That drove the proposal of the "ProfiledSolidDefinitionElement" name. The other one, being a concrete class meaning to be the default for 90% of the cases, we went with a shorter/cleaner name.